knowledge-base/actions/benc-uk/workflow-dispatch/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Workflow Dispatch'
```

Comment:
@vandana41 Please remove this file [benc-uk/workflow-dispatch/action-security.yml](https://github.com/step-security/secure-workflows/pull/1333/files#diff-f25155e0816dd0aa3322881ba9e3ead82d0f92562b67ae1d0734c7ca8346ac68) and also this file [svenstaro/upload-release-action/action-security.yml](https://github.com/step-security/secure-workflows/pull/1333/files#diff-bebe75251abe0f282aef9f8ab664b3160ccf81263faf52e46373123e6227077d) as you have already added these files in the previous pr ([here](https://github.com/step-security/sec1ure-workflows/pull/1322/files#diff-f25155e0816dd0aa3322881ba9e3ead82d0f92562b67ae1d0734c7ca8346ac68))
As it may result in conflicts while merging in future.